polaris-tokens/src/themes/base/color.ts

```diff
@@ -1138,4 +1140,10 @@ export const color: {
   'color-video-thumbnail-play-button-text-on-bg-fill': {
     value: colors.gray[1],
   },
+  'color-badge-new-bg-fill': {
+    value: colors.teal[5],
+  },
+  'color-badge-new-text': {
+    value: colors.teal[14],
+  },
```

Comment:
Not blocking but I noticed that this file appears to be in alpha order. If we want to preserve that I think these two new values should be listed after `color-backdrop-bg`.